proctl/proctl_linux_amd64.go

```diff
@@ -109,13 +109,17 @@ func (dbp *DebuggedProcess) findExecutable() (*elf.File, error) {
 func (dbp *DebuggedProcess) parseDebugFrame(exe *elf.File, wg *sync.WaitGroup) {
 	defer wg.Done()
 
-	debugFrame, err := exe.Section(".debug_frame").Data()
-	if err != nil {
-		fmt.Println("could not get .debug_frame section", err)
+	if sec := exe.Section(".debug_frame"); sec != nil {
+		debugFrame, err := exe.Section(".debug_frame").Data()
+		if err != nil {
+			fmt.Println("could not get .debug_frame section", err)
+			os.Exit(1)
+		}
+		dbp.FrameEntries = frame.Parse(debugFrame)
+	} else {
+		fmt.Println("No debug symbols found")
```

Comment:
Could you change this to something like "Error: Executable does not contain .debug_frame section" or something along those lines, just to make it more clear exactly why we're bailing.